service-account/bound-sa-token.md

```diff
@@ -1,122 +0,0 @@
-%title: Bound Service Account Token
-%author: Krzysztof Ostrowski
-%date: 10.03.2023
-
--> # Bound Service Account Token <-
-
---------------------------------------------------
--> # Table of Contents <-
-==============
-
--> # What is a Service Account Token  <-
--> # Service Account Token as a YAML <-
--> # Old Service Account Token Behavior  <-
--> # What is a Bound Service Account Token  <-
--> # Lord of Tokens: Kubelet <-
--> # Lord of ServiceAccounts: Service Account Admission Controller <-
--> # Bound Service Account Token in use by a Pod <-
-
--------------------------------------------------
--> # What is a Service Account Token  <-
-
-An **identity** that an instance or an application can use to make **API requests**:
```

Comment:
At my university, they taught us to be specific when describing something.

Service Account Token is not an identity. It's a _JWT representing an identity_. It serves the application to _authenticate_ itself in its API requests. It can make API requests even without the token.

Reply:
Sorry, I meant "Service Account". Yes, the token is a JWT representing and identity.